NMS-14382 Add documentation for OPAQUE data type support (#4904)
OpenNMS/opennms: docs/modules/operation/pages/performance-data-collection/data-types.adoc

== Opaque

> OpenNMS supports at the time only float numbers encoded as opaque. The floating point values (like gauges type) reflect
> the current state of a given metric.

Reviewer: ```suggestion
{page-component-title} only supports opaque values that encode floating point numbers.
The floating point values (like gauges type) reflect the current state of a given metric.
```
